models/llama/llama/math_ops.py

```diff
@@ -70,14 +71,15 @@ def attention(self, xq, keys, values, head_dim, mask):
 
     @Profiler.profiling_decorator("softmax")
     def softmax(self, x, dim):
-        if self.use_triton:
-            return F.softmax(x, dim=-1)
+        if self.use_triton and len(x) == 2:
+            return triton_softmax(x, dim=-1)
         else:
             return F.softmax(x, dim=-1)
 
     @Profiler.profiling_decorator("argmax")
     def argmax(self, x, dim):
         if self.use_triton:
+            # TODO: change
```

Comment:
Instead of adding a `TODO` to the code here, would you mind creating an issue to track it?

Reply:
removed